include/osquery/ev2/buffered_subscription.h

```diff
@@ -0,0 +1,113 @@
+/**
+ *  Copyright (c) 2018-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed as defined on the LICENSE file found in the
+ *  root directory of this source tree.
+ */
+
+#pragma once
+
+#include <osquery/ev2/event.h>
```

Comment:
ev2 was sort of the name for the event framework, not necessarily a version, but naming right :D ... This is going to be a completely different framework and both will work simultaneously for some time since it isn't feasible to port everything over in one go. So I separated both frameworks into 2 different directories to make it clear what is what, and make it easier for people to port things over time.

I created a completely new directory because both will work simultaneously for some time, since it isn't feasible to just port everything to the new framework at the same

Reply:
I understood that, I just don't think that `ev2` should be a long-lived directory name in the headers.